Add relative to permittivity and permeability

ClemensLinnhoff opened the pull request:
## Describe your changes

- Add "relative" to permittivity and permeability
- Add equations

The documentation now looks like this:

![image](https://github.com/user-attachments/assets/5492591c-3e30-4bed-88b5-ca43dd7be7c8)

![image](https://github.com/user-attachments/assets/d5b4b60f-894c-4ec5-b89a-669991839978)

Edit: As suggested by @lyndyRott, I changed n'' to kappa:
![image](https://github.com/user-attachments/assets/53640dce-54c9-48a1-b05d-b7df8f623076)


## Issue ticket number and link

Fixes #370, fixes #391

## Checklist before requesting a review

- [x] I have performed a self-review of my code/documentation.
- [x] My changes generate no new warnings during the documentation generation.

ClemensLinnhoff commented:
@SandroReith is this correct in you point of view?

AsamDiegoSanchez reviewed (approved):
Text reviewed
Signed Off by: Diego Sánchez <diego.sanchez@asam.net>

DavidJRitter904 commented:
I have added "relative" in the checker rules for consistency.

ClemensLinnhoff commented:
> I have added "relative" in the checker rules for consistency.

Good catch! But the validity range of the relative permeability based on mu_0 does not make sense, right? That would only apply to the absolute permeability.

DavidJRitter904 commented:
You are right! 
There was my consistency thinking stronger.
I have reverted my commit.

ClemensLinnhoff commented:
> > You are right! There was my consistency thinking stronger. I have reverted my commit.
> 
> But we still need the minimum and maximum values. Could you provide these boundaries for the relative values @DavidJRitter904?

I was working on this PR anyways, so I also fixed the min and max values in the checker rules. Does this work for you @DavidJRitter904 ?

lyndyRott commented on schemas/material_optical_schema.json:
Often, the imaginary part of the complex index of refraction is referred to as kappa.

$$ n = n' -j \kappa$$

We might want to switch $n''$ by $\kappa$ to be closer to general notation?!

ClemensLinnhoff replied:
You are right, I changed it.

DavidJRitter904 commented on content/09_annexes/checker-rules.adoc:
```diff
@@ -192,19 +192,25 @@ Description:: The humidity in percent (%) shall be within the range of 0 to 100.
 UID:: asam.net:xom-mat:1.0.0:xompt.real_part_permeability_valid_range
 
-Description:: The real part of permeability shall be within the range of 1.256637e-06 to 1.256637 (µ~0~ to µ~0~ x 10e06).
+Description:: The real part of the relative permeability shall not be below 1.
```
FYI, diamagnetic materials can have a real part which is slightly below 1. 
I can agree that this can be rounded to 1.

ClemensLinnhoff replied:
So, should we leave it at 1 or should we set the minimum to a lower value? Or to 0?

DavidJRitter904 replied:
Bismuth is an example below 1 as it has a value of 0.99983.
We can leave it at 1 until there might be some feedback which would require a value below 1.

DavidJRitter904 reviewed (commented):
Except of the one remark for diamagnetic material, this looks good for me.
